docs/source/amp.rst

```diff
@@ -0,0 +1,365 @@
+.. role:: hidden
+    :class: hidden-section
+
+Automatic Mixed Precision package - torch.amp
+==================================================
+
+.. automodule:: torch.amp
+.. currentmodule:: torch.amp
+
+``torch.amp`` provides convenience methods for mixed precision:  using fast FP16 arithmetic where you can and
+stable FP32 arithmetic where you must to optimize speed while maintaining accuracy and stability.
+
+The two ingredients of the mixed precision recipe are `Autocasting`_ and `Gradient Scaling`_.
```

Comment:
I think you should cut this section through the start of "Autocasting." Although since that section's "Under construction..." that might need to be cut, too.